integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -38,6 +38,7 @@
  */
 public class Hooks {
 
+    private static final String webhookId = Utils.getSystemProperty("webhook.site.uuid");
```

Comment:
Please use capital letters for constant.

Reply:
Okay!